torchvision/prototype/datapoints/__init__.py

```diff
@@ -1,5 +1,6 @@
 from ._bounding_box import BoundingBox, BoundingBoxFormat
 from ._datapoint import FillType, FillTypeJIT, InputType, InputTypeJIT
+from ._datapoint import FillType, FillTypeJIT, InputType, InputTypeJIT  # TODO: these may not need to be public?
```

Comment:
Most likely not. We only introduced them for internal convenience. Whenever you see something annotates with `*JIT`, you can basically look up the eager counterpart to see what is actually supported.